pkgs/applications/science/astronomy/gildas/wrapper.patch

```diff
@@ -0,0 +1,18 @@
+diff --new-file -r -u gildas-src-feb17d.orig/admin/wrapper.sh gildas-src-feb17d/admin/wrapper.sh
+--- gildas-src-feb17d.orig/admin/wrapper.sh	1970-01-01 01:00:00.000000000 +0100
++++ gildas-src-feb17d/admin/wrapper.sh	2017-05-18 21:00:01.660778782 +0200
```

Comment:
will this wrapper part in future versions of the software. Just asking because of the git-like hash in the patch path name.